fs/copy_nondarwin.go

```diff
@@ -0,0 +1,22 @@
+//go:build !darwin
+// +build !darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package fs
+
+var copyFile = openAndCopyFile
```

Comment:
I'm not wild about a `var` here vs `func copyFile(target, source string) error { return openAndCopyFile(target, source) }` but since it's private it's a very minor concern.

Reply:
Yeah me either, it just seemed cleaner though. If there is an argument to be made that var is slower, then will update.